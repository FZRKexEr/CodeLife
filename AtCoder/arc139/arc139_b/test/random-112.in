100
45 7 9 16 14 11
4 16 14 20 2 16
27 12 9 10 8 11
75 10 20 9 2 11
25 17 6 9 13 14
62 9 10 19 8 1
37 18 18 3 17 6
40 8 6 11 7 17
75 3 17 6 12 2
93 1 8 10 13 12
36 18 9 18 20 5
48 5 7 1 12 11
27 9 11 2 17 9
32 15 1 10 8 9
4 9 10 8 4 5
52 3 19 6 20 15
78 17 15 9 13 12
67 2 18 2 9 6
60 1 12 5 6 5
91 2 12 15 14 1
92 17 6 2 14 14
61 4 11 1 13 4
30 3 20 1 13 20
79 15 8 9 14 17
57 5 1 9 17 11
97 16 15 5 12 7
84 17 14 3 19 14
41 16 1 2 10 6
54 14 16 9 3 3
76 7 8 14 16 15
5 6 5 15 3 8
96 15 16 9 14 5
88 15 18 8 20 8
53 20 17 6 19 4
83 16 18 8 19 3
85 12 8 18 19 3
6 16 10 12 15 14
55 3 18 2 9 7
81 8 17 3 15 1
76 1 14 19 10 11
23 10 11 10 11 3
53 10 11 12 6 10
34 8 9 2 17 8
78 18 17 1 8 10
53 20 3 18 17 20
18 6 19 5 12 16
31 10 1 12 12 20
99 8 14 6 5 5
97 17 18 17 15 18
69 6 16 13 17 14
2 7 7 8 8 18
45 10 17 1 7 13
76 14 20 3 1 10
87 11 12 6 11 11
4 17 14 11 16 3
14 16 8 3 5 18
93 12 11 18 6 5
54 7 2 16 9 19
37 13 6 14 12 7
89 6 5 9 14 4
72 15 4 11 9 7
77 5 18 20 14 2
48 20 8 11 15 18
73 17 8 18 14 1
9 18 3 4 8 16
39 18 8 11 2 20
52 18 6 2 15 8
14 15 12 4 20 18
41 8 6 14 3 16
36 6 2 20 20 5
39 12 2 1 16 8
1 6 11 17 18 10
18 16 13 19 15 19
5 13 5 12 3 1
94 17 3 15 6 11
91 9 19 5 17 3
61 15 17 8 18 16
4 17 18 6 18 2
50 7 13 17 5 1
75 8 11 17 13 10
67 13 19 9 8 12
62 7 11 9 17 18
4 2 1 1 4 9
4 2 8 5 13 18
94 7 2 18 8 20
65 19 1 6 2 14
28 15 17 11 12 4
83 19 11 20 3 15
24 17 3 13 14 19
90 19 7 12 4 15
91 13 12 1 9 3
1 16 11 16 11 8
39 19 6 16 14 13
39 16 15 8 6 7
40 14 16 3 4 3
9 9 9 13 19 4
56 7 19 14 14 4
48 5 5 18 6 16
63 8 5 2 10 19
17 17 18 4 14 11
