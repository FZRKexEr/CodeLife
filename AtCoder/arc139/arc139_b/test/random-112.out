55
80
33
59
45
44
15
35
38
182
20
48
28
34
32
114
70
44
25
64
84
26
30
106
205
77
84
38
63
154
8
30
60
24
87
64
72
23
34
115
16
42
38
38
132
36
48
41
209
121
16
33
14
95
44
36
45
117
40
104
57
22
48
21
36
37
54
26
29
90
39
17
53
1
63
59
80
24
52
90
58
107
4
20
142
54
125
86
65
46
28
16
44
68
30
4
92
108
84
14
